net/node/transactionPool.go

```diff
@@ -14,88 +16,365 @@ import (
 
 type TXNPool struct {
 	sync.RWMutex
-	txnCnt uint64
-	list   map[common.Uint256]*transaction.Transaction
+	txProcessList     // transaction which have been verifyed will put into this map
+	tXNPendingList    // transaction which didn't pass the verify will put into this map
```

Comment:
1. txProcessList  is the origin txlist.
2. PendingList is used to store all the tx which didn't pass the validation check. which can tell the user(sdk)  failed reason and which need to  redo or need other operations.
3. assetIssueSummary  is used to store all the issued  amount  in  transaction pool ,  so the new tx only need to check the delta amount instead of check all the tx in pool. it can reduce the process time.
4. inputUTXOList like the #3 ,instead of check all the tx in pool ,only need to check the delta of new tx.